examples/backbone_marionette/js/TodoMVC.Application.js

```diff
@@ -0,0 +1,27 @@
+/*global Backbone, TodoMVC:true */
+
+var TodoMVC = TodoMVC || {};
+
+(function () {
+	'use strict';
+
+	var TodoApp = Backbone.Marionette.Application.extend({
```

Comment:
heh we *could* just use an object instead of an app since app is really pointless currently ;)